libretro-common/streams/file_stream.c

```diff
@@ -176,6 +177,37 @@ RFILE* filestream_open(const char *path, unsigned mode, unsigned hints)
    return output;
 }
 
+RFILE* filestream_open_child(RFILE *stream, const char* path)
```

Comment:
this is the new method - pass a specific filename to open a file within a track/bin or "TRACK NN" to open a track within a CUE or CDROM parent filestream.